Client/qtTeamTalk/statusbardlg.cpp

```diff
@@ -51,9 +61,100 @@ void StatusBarDlg::slotStatusBarEventToggled(const QModelIndex &index)
     auto events = m_statusbarmodel->getStatusBarEvents();
     StatusBarEvent e = StatusBarEvent(index.internalId());
     if (e & events)
+    {
         m_statusbarmodel->setStatusBarEvents(events & ~e);
+        ui.sbmsg_groupbox->hide();
+    }
     else
+    {
         m_statusbarmodel->setStatusBarEvents(events | e);
+        ui.sbmsg_groupbox->show();
+    }
+}
+
+void StatusBarDlg::statusBarEventSelected(const QModelIndex &index)
+{
+    saveCurrentMessage();
+
+    m_currentIndex = index;
+    if (!index.isValid()) return;
+
+    bool customizable = true;
+    auto eventMap = UtilUI::eventToSettingMap();
+    StatusBarEvents eventId = static_cast<StatusBarEvents>(index.internalId());
+
+    if (eventMap.contains(eventId))
+    {
+        const StatusBarEventInfo& eventInfo = eventMap[eventId];
+        QString paramKey = eventInfo.settingKey;
+        QString defaultValue = UtilUI::getDefaultValue(paramKey);
+        QString currentMessage = ttSettings->value(paramKey, defaultValue).toString();
+        if (eventInfo.eventName.size() > 0)
```

Comment:
When "subscription change" and "classroom change" events are called then this label is set to the event message. Why this behaviour? Also the label becomes too long. Shouldn't this just be removed?

Reply:
I think it's more clear to understand that the message for each subscription change or classroom is the same, without this behaviour I didn't find really clear the fact that each subscription or each classroom change available in list don't have his own message.
Maybe we can just choose a shorter event name?
We also have the same behaviour in TTS tab so if a change is needed we'll have to do the same in TTS tab also

Reply:
The problem is that the "Message for Event" stays. There should be an "else" to reset to default behaviour.